bin/compile/function_metadata/metadata_builder.dart

```diff
@@ -0,0 +1,180 @@
+//  Copyright 2021 Abitofevrything and others.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+import 'package:analyzer/dart/ast/ast.dart';
+import 'package:analyzer/dart/constant/value.dart';
+import 'package:analyzer/dart/element/element.dart';
+import 'package:analyzer/src/dart/element/element.dart';
+import 'package:nyxx_commands/nyxx_commands.dart';
+
+import '../generator.dart';
+import '../type_tree/tree_builder.dart';
```

Comment:
please replace relative improrts with library imports

Reply:
The imports are relative in the entire rest of the package following [Dart style guidelines](https://dart.dev/guides/language/effective-dart/usage#prefer-relative-import-paths) so they are relative here too.